.github/workflows/semgrep.yml

```
@@ -27,7 +27,13 @@ jobs:
           [[ "${{ inputs.verbose }}" == "true" ]] && export VERBOSE=--verbose || export VERBOSE=''
           echo "VERBOSE_OPTION=$VERBOSE" >> $GITHUB_ENV
       - name: Scan
-        run: semgrep ci ${{ env.VERBOSE_OPTION }}
+        run: |
+          semgrep ci ${{ env.VERBOSE_OPTION }}
+          BBlue='\033[1;34m'
+          Color_Reset='\033[0m'
+          echo ""; echo -e "📝 ${BBlue}View results ${{ github.repository }} in Semgrep Cloud Platform${Color_Reset}"
+          echo "  https://semgrep.dev/orgs/-/findings?repo=Refinitiv%2Frefinitiv-ui"
+          echo "  https://semgrep.dev/orgs/-/supply-chain?q=refinitiv-ui"
```

this link add filtering to our main branches only.
```suggestion
          echo "https://semgrep.dev/orgs/-/supply-chain?q=Refinitiv%2Frefinitiv-ui&ref=refs%2Fheads%2Fv7%2Cv6"
```

---

I have updated links by filtering the repository only.